apps/web/payload.config.ts

```diff
@@ -233,6 +265,18 @@ export default buildConfig({
     }
   },
   secret: process.env.PAYLOAD_SECRET || '',
+  email: nodemailerAdapter({
```

Comment:
lets add some documentation on the form plugin and the email adapter (how to configure with a thrid-party mailer) into the README